packages/cloud_firestore/lib/src/query.dart

```diff
@@ -126,6 +126,8 @@ class Query {
     dynamic isGreaterThan,
     dynamic isGreaterThanOrEqualTo,
     dynamic arrayContains,
+    dynamic arrayContainsAny,
+    dynamic whereIn,
```

Comment:
arrayContainsAny and whereIn should be `List<dynamic>`

Reply:
@slightfoot Thanks! Changed it.